admin_guide/high_availability.adoc

```diff
@@ -75,66 +196,67 @@ The list of command options described here are a subset that are relevant to thi
-An example configuration for running a
-https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[geo-cache]
-service is provided along with this document. The docker image referenced in
-that file `myimages/geo-cache` does not exist and is there purely for reference
-purposes. Edit the
-https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[geo-cache.json]
-configuration to replace `myimages/geo-cache` with your favorite cache software
-docker image, change the number of replicas to 4 and ensure the label matches
-the one used in step 1 above `ha-cache=geo`.
-
-****
-`$ osc create -n <your-namespace-here> -f ./examples/geo-cache.json`
-****
-
-===== Step 3: Configure IP failover for the geo cache service
-The final step is to configure the virtual IPs and failover for the nodes labelled in step 1 (with "ha-cache=geo"). Ensure the number of replicas matches the number of nodes that satisfy the constraint or label we used in step 1.  Specify the virtual IP address and the port number 9736 that IP failover should monitor on those instances.
-
-****
-`$ osadm ipfailover ha-geo-cache --replicas=4 --selector="ha-cache=geo" --virtual-ips=10.245.2.101-104 --watch-port=9736 --credentials="$OPENSHIFTCONFIG" --create`
-****
-
-==== Internals
-The osadm ipfailover command ensures that an ipfailover sidecar pod runs on each of the nodes matching the constraints or label we used in step 1. This ipfailover sidecar pod uses VRRP (Virtual Router Reduncy Protocol) within keepalived to monitor that the service on the watched port is available and keepalived will automatically float the VIPs in event of the service not being available.
-
-In our example, we can now use the VIPs 10.245.2.101 through 10.245.2.104 to send traffic to the geo-cache service. If a particular geo-cache instance is "unreachable" (as an example due to a node failure), keepalived will float the VIPs automatically float amongst group of nodes we labelled "ha-cache=geo" and the service would still be reachable via those virtual IP addresses.
-
-== Examples
-
-See what the IP failover configuration would look like if it is created:
-
+////
+
+=== Setting up an IP-based Network Service
+The following steps describe how to set up a highly-available IP-based network
+service with IP failover:
+
+. Label the nodes for the service. This step can be optional if you run the
+service on any of the nodes in your Kubernetes cluster and use VIPs that can
+float within those nodes. This process may already exist within a complex
+cluster, in that the nodes may be filtered by any constraints or requirements
+specified (e.g. nodes with SSD drives, or higher cpu/memory/disk requirements,
+etc.).
++
+The following example labels a highly-available cache service that is
+binding/listening on port 9736 as "ha-cache=geo":
++
+====
+----
+$ openshift kube label nodes openshift-minion-{6,3,7,9} "ha-cache=geo"
```

Comment:
https://github.com/openshift/openshift-docs/pull/332 has merged, so you could update this to:

```
$ osc label nodes openshift-minion-{6,3,7,9} "ha-cache=geo"
```